src/components/chip-set/chip-set.scss

```diff
@@ -184,6 +184,7 @@ limel-icon.mdc-chip__icon.mdc-chip__icon--leading {
 
     &.disabled:not(.mdc-chip-set--input) {
         @include shared_input-select-picker.looks-disabled;
+        pointer-events: none;
```

Comment:
First I wanted to refactor some stuff in the chip-set, as we are disgustingly handling the `disabled` stated on chips of different types, using CSS hacks and `tabindex` hacks. However, since we are gonna refactor the chip-set and use `limel-chip` hopefully soon, this won't be an issue in future. So I'm just recreating what we used to get from the mixin here instead.